pom.xml

```diff
@@ -9,7 +9,7 @@
         <version>2.0.1</version>
     </parent>
     <properties>
-        <eiffel-remrem-publish.version>2.0.2</eiffel-remrem-publish.version>

```

Comment:
The version 2.0.3 is already released. I have merged my changes yesterday. Please fetch the latest changes and changes the version.

Reply:
fixed.